app/code/Magento/AdvancedPricingImportExport/Model/Import/AdvancedPricing/Validator/Website.php

```diff
@@ -63,6 +61,7 @@ protected function isWebsiteValid($value, $websiteCode)
      * Validate value
      *
      * @param mixed $value
```

Comment:
Would you mind to replace `mixed` with the appropriate type?

Reply:
**Done**